### PR TITLE
docs: added blog `Get Front-End Test Coverage with Cypress` 

### PR DESCRIPTION
**Pull request by guoqqqi**
Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

Added a blog "Get Front-End Test Coverage with Cypress".

### CODE REVIEW

File: website/blog/2021-03-02-get-front-end-test-coverage-with-cypress.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## Installation Dependencies & Configuration
+
+To collect test coverage data, we need to put some probes in the original business code for Cypress to collect the data.
+
+Cypress officially recommends two approaches, the first is to generate a temporary directory via `nyc` and run the code that has been written to the probe to collect test coverage data. The second way is to do the code conversion in real time through the code conversion pipeline, which eliminates the hassle of temporary folders and makes collecting test coverage data relatively refreshing. We choose the second way to collect front-end E2E coverage.
+
+1. Installing Dependencies
+
+    yarn add  babel-plugin-istanbul --dev
```
Comment (LiteSun) [inline website/blog/2021-03-02-get-front-end-test-coverage-with-cypress.md]:
```suggestion
```
    yarn add  babel-plugin-istanbul --dev
```
```

File: website/blog/2021-03-02-get-front-end-test-coverage-with-cypress.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+2. Install the cypress plug-in
+
+    yarn add  @cypress/code-coverage --dev
```
Comment (LiteSun) [inline website/blog/2021-03-02-get-front-end-test-coverage-with-cypress.md]: ditto.

File: website/blog/2021-03-02-get-front-end-test-coverage-with-cypress.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+5. Get Test Coverage
+
+After the configuration is done, we need to run the test case. After the test case is run, Cypress will generate `coverage` and
+`.nyc_output` folders, which contain the test coverage reports.
+
+![1.png](https://lh4.googleusercontent.com/o-tyQagmCjprpNkuTjMFLaALZKtW4pyC9nj-GcPx4MM3xK0zrMED9Nndk5ZmZkZsQ5SIJPEovcrHyjWP2YXtEcYYDpLL49aV_97N83doTkOuMXlFsVjGu53A9FdlxOCr6i3aIDTA)
+
+The test coverage information will appear in the console after executing the following command.
+
+    npx nyc report --reporter=text-summary
```
Comment (LiteSun) [inline website/blog/2021-03-02-get-front-end-test-coverage-with-cypress.md]: ditto.

File: website/blog/2021-03-02-get-front-end-test-coverage-with-cypress.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+> [@LiteSun](https://github.com/LiteSun), Apache APISIX Committer from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+> Source:
+>
+> - https://github.com/apache/apisix
+> - https://github.com/apache/apisix-dashboard
+
+## Background
+
+In the article ["Stable Product Delivery with Cypress"](/blog/2021/02/08/stable-product-delivery-with-cypress), we discussed why we chose Cypress as our E2E testing framework. After spending nearly two months refining the test cases, we needed test coverage to quantify whether the test coverage was sufficient. This article will describe how to use Cypress to get E2E coverage on the front end of APISIX Dashboard.
```
Comment (LiteSun) [inline website/blog/2021-03-02-get-front-end-test-coverage-with-cypress.md]:
```suggestion
In the article ["Stable Product Delivery with Cypress"](/blog/2021/02/08/stable-product-delivery-with-cypress), we discussed why we chose Cypress as our E2E testing framework. After spending nearly two months refining the test cases, we needed test coverage to quantify whether the test coverage was sufficient.This article will describe how to get APISIX Dashboard front-end E2E coverage using Cypress.
```

**Review by LiteSun — CHANGES_REQUESTED** [review]

**Review by LiteSun — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**